test/e2e/persistent_volumes.go

```diff
@@ -374,17 +400,22 @@ func testPodSuccessOrFail(c clientset.Interface, ns string, pod *api.Pod) {
 
 // Delete the passed in pod.
 func deletePod(f *framework.Framework, c clientset.Interface, ns string, pod *api.Pod) {
+	if c != nil {
+		if pod != nil && len(pod.Name) > 0 {
+			framework.Logf("Deleting pod %v", pod.Name)
+			_, err := c.Core().Pods(ns).Get(pod.Name)
```

Comment:
What are we testing here? Is it volume controller that binds+recycles/deletes persistent volumes or API server machinery? Let's trust API server ff we're testing PV controller. IMO API server tests belong somewhere else.